README.md

```diff
@@ -115,6 +115,10 @@ Maps the specified path parameter `name` to a specialized param-capturing middle
 
 This function positions the middleware in the same stack as `.use`.
 
+If a `Promise` object is returned from the `param_middleware` function, the router
+will attach an `onRejected` callback using `.then`. If the promise is rejected,
```

Comment:
```suggestion
will attach an `onRejected` callback using `.then(null, onRejected)`. If the promise is rejected,
```

I only think this could use more explicit call out because there is no commonly agreed upon wording I have seen other places.  Having the full explicit call might help make it more clear.